cfgov/unprocessed/css/pages/ask.less

```diff
@@ -3,9 +3,11 @@
    Ask CFPB pages at /ask-cfpb/.
    ========================================================================== */
 
+@readable-width: 600px;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2922/commits/74bd6258ad4b99a2c2833f73e4042571e8eee3f1